src/api.ts

```diff
@@ -28,6 +28,11 @@ const DISCORD_COUNTER = 'mirrord-discord-counter';
 */
 const DISCORD_COUNTER_PROMPT_AFTER = 10;
 
+/**
+* Environment variable name for listing targets with a specific type via the CLI 'ls' command.
+*/
+const LS_TARGET_TYPES_ENV = "MIRRORD_LS_TARGET_TYPES";
```

Comment:
```suggestion
const MIRRORD_LS_TARGET_TYPES = "MIRRORD_LS_TARGET_TYPES";
```
Why not have the same name, or maybe `MIRRORD_LS_TARGET_TYPES_ENV` have the `_ENV` at the end too. Mega nit.